commons/connector/fitbit/fitbit_activity_heart_rate.avsc

```diff
@@ -0,0 +1,18 @@
+{
+  "namespace": "org.radarcns.connector.fitbit",
+  "name": "FitbitActivityHeartRate",
+  "type": "record",
+  "doc": "Aggregate heart rate as measured over an activity. The heart rates are divided amongst so-called heart rate zones. A heart rate zone is a range of heart rates and it does not overlap with other heart rate zones. These zones are called, in order of increasing heart rate, 'Out of Range', 'Fat Burn', 'Cardio', and 'Peak'.",
```

Comment:
do we want to qualify these zones explicitly https://blog.fitbit.com/heart-rate-zones/? Even if they may change in the future the zones or bins.

Reply:
on second thoughts perhaps better left in Fitbit docs.

Reply:
Ah I already updated the docs, is the following ok?
> Aggregate heart rate as measured over an activity. The heart rates are divided amongst so-called heart rate zones. A heart rate zone is a range of heart rates and it does not overlap with other heart rate zones. These zones are called, in order of increasing heart rate, 'Out of Range', 'Fat Burn', 'Cardio', and 'Peak'. Fitbit uses the common formula of 220 minus your age to estimate your maximum heart rate. The 'Out of Range' zone has heart rates up to 50 percent of maximum heart rate, the 'Fat Burn' has heart rates up to 70 percent of maximum heart rate, 'Cardio' up to 85 percent of maximum heart rate and the 'Peak' range has heart rates higher than that.